lib/cluster.rb

```diff
@@ -106,7 +106,7 @@ def self.for_ocns(ocns)
     dataset = table
       .select(:cluster_id)
       .distinct
-      .where(ocn: ocns)
+      .where(ocn: ocns.to_a)
```

Comment:
Sequel doesn't like it when given a `Set`; might as well coerce to an array rather than making the caller do it.